docs/network_firewall.rst

```diff
@@ -578,3 +582,25 @@ Once it is complete, you will see a notification of successful upgrade:
 .. [#] Tails screenshots were taken on Tails 3.0~beta4. Please make an issue on
        GitHub if you are using the most recent version of Tails and the
        interface is different from what you see here.
+
+.. _abstract firewall rules:
+
+Abstract firewall rules
+-----------------------
+
+The pfSense instructions using the web interface can also be precisely
+described as follows:
+
+* Disable DHCP (in case the firewall is providing a DHCP server by default)
+* Disallow all traffic by default (inbound or outbound)
+* Allow UDP OSSEC (port 1514) from *Application Server* to *Monitor Server*
+* Allow TCP ossec agent auth (port 1515) from *Application Server* to *Monitor Server*
+* Allow TCP/UDP DNS from *Application Server* and *Monitor Server* to the IP of know name servers
+* Allow UDP NTP from *Application Server* and *Monitor Server* to all
+* Allow TCP any port from *Monitor Server* to all
+* Allow TCP 80/443 from *Application Server* and *Monitor Server* to all
```

Comment:
TCP 80/443 is _only_ needed for the admin workstation (to access the pfSense web interface), otherwise this should be disabled (a common misconfiguration is TCP 80/443 inbound to the application and monitor servers because people forget that all the webserver traffic is over Tor onion services)

Reply:
I changed to ```Allow TCP 80/443 from *Admin Workstation* to all (in case there is a need to access the web interface of the firewall)``` is that better ?